server/src/test/java/org/elasticsearch/cluster/metadata/IndexMetadataVerifierTests.java

```diff
@@ -118,11 +118,8 @@ public void testIncompatibleVersion() {
                     + "] "
                     + "but the minimum compatible version is ["
                     + minCompat
-                    + "]."
-                    + " It should be re-indexed in Elasticsearch "
-                    + (Version.CURRENT.major - 1)
-                    + ".x before upgrading to "
-                    + Version.CURRENT
+                    + "]. It should be re-indexed using the previous major version of Elasticsearch before upgrading to "
```

Comment:
My comment in `IndexMetadataVerifier` applies here too: we could call `Build.current().major()` and get an `8` for our message.

Reply:
As per the other conversation, we agreed to leave the message as-is until we have a better understanding how major will evolve (also in light of CD projects)